.gitignore

```diff
@@ -0,0 +1,4 @@
+*~
+*.swp
+__pycache__
+fx_crash_sig.egg-info/
```

Comment:
`.gitignore` helps deal with random files that shouldn't get checked in. I threw things that affected me in here.